libmattermost.c

```diff
@@ -4755,6 +4772,16 @@ plugin_load(PurplePlugin *plugin, GError **error)
 						PURPLE_CMD_FLAG_PROTOCOL_ONLY | PURPLE_CMD_FLAG_ALLOW_WRONG_ARGS,
 						MATTERMOST_PLUGIN_ID, mm_cmd_topic,
 						_("topic <description>:  Set the channel topic description"), NULL);
+
+	purple_cmd_register("header", "s", PURPLE_CMD_P_PLUGIN, PURPLE_CMD_FLAG_CHAT |

```

Comment:
no such commands in mattermost: but header and purpose are set in Channel Menu -> Edit Channel Header / Edit Channel purpose.